bazel/external/ubuntu_packages/packages.bzl

```diff
@@ -17,6 +17,8 @@
 # This file is autogenerated, do not edit directly.
```

Comment:
Since the upgrade this has become an autogenerated file, so instead of editing it please edit the `debs.json` file.

Reply:
Whoops, I missed that. I'll update the right file.